plotly/tools.py

```diff
@@ -916,11 +954,13 @@ def _add_domain(layout, x_or_y, label, domain, anchor, position):
         if isinstance(position, float):
             axis['position'] = position
         layout[name] = axis
+        list_of_domains.append(domain)  # added for subplot titles
 
     # Function pasting x/y domains in layout object (3d case)
     def _add_domain_is_3d(layout, s_label, x_domain, y_domain):
         scene = graph_objs.Scene(domain={'x': x_domain, 'y': y_domain})
         layout[s_label] = scene
+        list_of_domains.append(domain)  # added for subplot titles
```

Comment:
@cldougl You'll have to change this in order to get the tests to pass.

As annotations don't work with 3d yet, I would simply skip 3d domains.